docs/timeseries/analysis.rst

```diff
@@ -0,0 +1,213 @@
+
+Time series can also be combined 'horizontally' or column-wise with other tables
+using the :func:`~astropy.table.hstack` function, though these should not be
+time series (as having multiple time columns would be confusing)::
+
+    >>> from astropy.table import Table, hstack
+    >>> data = Table(data={'temperature': [40., 41., 40., 39., 30.] * u.K})
+    >>> ts_a_data = hstack([ts_a, data])
+    >>> ts_a_data
+    <TimeSeries length=5>
+              time            flux  temperature
+                              mJy          K
+             object         float64    float64
+    ----------------------- ------- -----------
+    2016-03-22T12:30:31.000     1.0        40.0
+    2016-03-22T12:30:34.000     4.0        41.0
+    2016-03-22T12:30:37.000     5.0        40.0
+    2016-03-22T12:30:40.000     3.0        39.0
+    2016-03-22T12:30:43.000     2.0        30.0
+
+Sorting time series
+===================
+
+Sorting time series in-place can be done using the
+:meth:`~astropy.table.Table.sort` method, as for |Table|::
+
+    >>> ts = TimeSeries(time='2016-03-22T12:30:31',
+    ...                        time_delta=3 * u.s,
+    ...                        data={'flux': [1., 4., 5., 3., 2.]})
+    >>> ts
+    <TimeSeries length=5>
+              time            flux
+             object         float64
+    ----------------------- -------
+    2016-03-22T12:30:31.000     1.0
+    2016-03-22T12:30:34.000     4.0
+    2016-03-22T12:30:37.000     5.0
+    2016-03-22T12:30:40.000     3.0
+    2016-03-22T12:30:43.000     2.0
+    >>> ts.sort('flux')
+    >>> ts
+    <TimeSeries length=5>
+              time            flux
+             object         float64
+    ----------------------- -------
+    2016-03-22T12:30:31.000     1.0
+    2016-03-22T12:30:43.000     2.0
+    2016-03-22T12:30:40.000     3.0
+    2016-03-22T12:30:34.000     4.0
+    2016-03-22T12:30:37.000     5.0
+
+Resampling
+==========
+
+We provide a :func:`~astropy.timeseries.simple_downsample` function
+that can be used to bin values from a time series into bins of equal time, using
+a custom function (mean, median, etc.). This operation returns a
+|BinnedTimeSeries|. Note that this is a simple function in the sense that it
+does not for example know how to treat columns with uncertainties differently
+from other values, and it will blindly apply the custom function specified to
+all columns.
+
+The following example shows how to use this to bin a light curve from the Kepler
+mission into 20 minute bins using a median function. First, we read in the data
+using:
+
+.. plot::
+   :context: reset
+   :nofigs:
+
+    from astropy.timeseries import TimeSeries
+    from astropy.utils.data import get_pkg_data_filename
+    example_data = get_pkg_data_filename('timeseries/kplr010666592-2009131110544_slc.fits')
+    kepler = TimeSeries.read(example_data, format='kepler.fits')
+
+(see :ref:`timeseries-io` for more details about reading in data). We can then
```

Comment:
This sentence should be folded into the narrative above because currently it looks weird in the rendered HTML because the plot directive above is a `:nofigs:`.

That said, I wonder if there is value is hiding the data loader snippet above. People don't like clicking extra link just to find out how to call those 4 lines.

Reply:
I'd say either this should be shown (i.e. not "nofigs") as @pllim suggests, or this paragraph about loading should be removed and something like ``the data from `:ref:`timeseries-io` are used here``

Reply:
Oh this was a mistake, I'm missing ``:include-source:``